jhove-core/src/main/java/edu/harvard/hul/ois/jhove/handler/XmlHandler.java

```diff
@@ -4355,21 +4355,21 @@ private void writeAESTimeRange (String baseIndent,
 	sampleAttrs[0][1] = "S" + Integer.toString ((int) sr);
 	_writer.println (margn2 + elementStart ("tcf:samples", sampleAttrs));
 	_writer.println (margn3 + element ("tcf:numberOfSamples",
-				   Integer.toString (start.getSamples ())));
+				   Long.toString (start.getSamples ())));
 	_writer.println (margn2 + elementEnd("tcf:samples"));
 	_writer.println (margn2 + element ("tcf:filmFraming", ffAttrs));
 	_writer.println (margn1 + elementEnd ("tcf:startTime"));
- 
+
 	if (duration != null) {
 	    _writer.println (margn1 + elementStart ("tcf:duration", attrs));
 	    _writer.println (margn2 + element ("tcf:hours",
-			       Integer.toString (duration.getHours ())));
+			       Long.toString (duration.getHours ())));
 	    _writer.println (margn2 + element ("tcf:minutes",
-			       Integer.toString (duration.getMinutes ())));
+			       Long.toString (duration.getMinutes ())));
 	    _writer.println (margn2 + element ("tcf:seconds", 
-			       Integer.toString (duration.getSeconds ())));
-	    _writer.println (margn2 + element 
-                    ("tcf:frames", Integer.toString (duration.getFrames ())));
+			       Long.toString (duration.getSeconds ())));
+	    _writer.println (margn2 + element ("tcf:frames",
+			       Long.toString (duration.getFrames ())));

```

Comment:
Mixed spacing by the looks

Reply:
Oh yeah, the *Handler files are a mess with white-space inconsistencies, even with tabs a spaces in the same line at times, or switching every handful of lines. I originally attempted to fix it, but decided in the end it wasn't worth my time to fix it all.

The half tabs and half spaces is actually consistent with the code's surrounding lines, which you can see by slowly highlighting line 4370.